dev_provisions/environment.sh

```diff
@@ -0,0 +1,16 @@
+export PROJECT_URL='http://192.168.68.68'
+export INSTALL_LOG='/tmp/install.log'
+export DEBIAN_FRONTEND=noninteractive
+export BLACKOPS_DATABASE_PASSWORD='ebwiki'
+export DATABASE_DUMP_FILE='09_2018_production.dump'
```

Comment:
Yes, this is hardcoded at the moment to reflect the file that's in the repo.   I use the environment file so only this one change has to be made to support a dump file with a different name.  It might be more flexible if the dump was named the same each time.